demo/nodejs/example.js

```diff
@@ -21,17 +31,34 @@ const transferData = {
     transaction_file: "/tmp/rgb-node/output/transaction"
 }
 
-ex.start_rgb(config.network, config.stash_endpoint, JSON.stringify(config.contract_endpoints), config.threaded,
-             config.datadir)
-    /*.then(r => ex.issue(r, JSON.stringify({
-        network: "testnet",
-        ticker: "USDT",
-        name: "USD Tether",
-        issue_structure: "SingleIssue",
-        allocations: [{ coins: 100, vout:0, txid: "0313ba7cfcaa66029a1a63918ebc426259f00953016c461663315d1bf6b83ab4" }],
-        precision: 0,
-    })))*/
-    .then(r => ex.transfer(r, JSON.stringify(transferData.inputs), JSON.stringify(transferData.allocate),
-                           transferData.invoice, transferData.prototype_psbt, transferData.consignment_file,
-                           transferData.transaction_file))
-    .catch(e => console.log(e))
+var runtime = null
+
+async function main() {
+    await ex.start_rgb(
+        config.network, config.stash_endpoint, JSON.stringify(config.contract_endpoints), config.threaded, config.datadir)
+    .then(r => {
+        runtime = r
+        return ex.issue(runtime, JSON.stringify(issueData))
+    })
+    /*
+    .then(() => {
+        return ex.transfer(runtime, JSON.stringify(transferData.inputs), JSON.stringify(transferData.allocate),
```

Comment:
Why we have commented out the transfer function from the demo?

Reply:
Anyway I think the correct place to add it back will be #21 so merging this version here

Reply:
@dr-orlovsky as reported on #14, currently `transfer` is not working, so for now it's better to leave it commented. I think we should add both `import` and `invoice` methods (both on `rgb::i9n` and in the sdk). if you agree on this I'll open the related issues

Reply:
Agree